src/_includes/head.html

```diff
@@ -0,0 +1,28 @@
+<html lang="en-gb">
```

Comment:
Could the `html` tag move too as that is what gets closed in a different partial?